### DISCUSSION TITLE
请问我下载了插件日历，但为什么它没有读取我之前的笔记日期？

### DISCUSSION THREAD

**Discussion by eveseed** [obsidianzh/forum#100] [Q&A常见问题]
小白提问：
最近下载了插件-calendar日历，但是插件并没有读取我以前的笔记的日期
而且我新建的笔记它也不会读取
只有我点击日历、插件端帮我建立的笔记，才会被这个日历插件读取（下方才有小点点）
非常困惑，请问各位大大，我是特例吗？

而且我的标题必须要保持建立时的标题，比如“2019-08-23”，如果我修改，日历处也看不到任何痕迹，求哪位大大给点提示。。。献花

**Comment by cyberovid** [comment]
日历可以在 插件选项-日记 修改特定的格式，calendar是第三方，日记是核心插件

**Reply by eveseed** [reply]
终于明白了，calendar这个功能是和日记这个功能一起使用的，现在我在日记里留下笔记，都会记录在calendar里啦，哈哈哈。
感谢大大赐教。
